src/Components/Form.js

```diff
@@ -34,7 +34,7 @@ const Form = () => {
     state.author = '';
   };
   return (
-    <div className=" mt-5 container">
+    <div className=" container">
```

Comment:
- [ ] Please update this placeholder from `Author` to `Category`
- [ ] Also add the arrow on that placeholder
![image](https://user-images.githubusercontent.com/62705947/186641279-a3ea694e-d7c0-4fcf-bbd9-6f1edca5cbe4.png)

Reply:
- [ ] Update the input borders and button, check the picture for more info :arrow_double_down: 
![image](https://user-images.githubusercontent.com/62705947/186642766-84b28d04-8ccf-4146-84de-4893897dab68.png)